data/json/mapgen/deserter_city/deserter_city_office.json

```diff
@@ -0,0 +1,317 @@
+        "/g````#hd.V..V..htth.fw`",
+        "/g````waf^V..V..htth..w`",
+        "/g````#####..G..htth..w`",
+        "/g````#...G..Va......^#`",
+        "/g````#hd.V..##########`",
+        "/g````waf.V..#.......a#`",
+        "/##########..#.ccccccc#`",
+        "/#fff#^......+........#`",
+        "/#...+.......#.....???#`",
+        "/########+#####VGGVVVV#`",
+        "/#T#T#T#yi#EEE#.......#`",
+        "/#+#+#+####EEE-......?w`",
+        "/#iiiii+..#EEH#......?#`",
+        "/#CSCSC#..#####......nw`",
+        "/#######.............?#`",
+        "/#CSCSC#..#####......?w`",
+        "/#iiiii+..#ZZZ#.......#`",
+        "/#+#+#+#..#ZZZ-..,##:##`",
+        "/#T#T#T#..#ZZZ#...#,.>#`",
+        "4#######ww#####ww,,#,##`",
+        "////////////////////////"
+      ],
+      "palettes": [ "deserter_city" ],
+      "terrain": { "g": "t_gutter", "`": "t_flat_roof", ":": "t_door_b" },
+      "place_npcs": [
+        { "class": "deserter", "x": 13, "y": 6 },
+        { "class": "deserter", "x": 9, "y": 19 },
+        { "class": "deserter", "x": 17, "y": 5 }
+      ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": "deserter_city_office_3f",
+    "object": {
+      "fill_ter": "t_linoleum_gray",
+      "rows": [
+        "////////////////////////",
+        "//////#################/",
+        "//////#iii*.......*iii#/",
+        "//////#iii%...c...%iii#/",
+        "//////#iii*...c...*iii#/",
+        "//////#****...c...****#/",
+        "//////#iii*...Q...*iii#/",
+        "//////#iii%...c...%iii#/",
+        "//////#iii*...c...*iii#/",
+        "/RRRRR#****...c...****#/",
+        "/Rssss#...............#/",
+        "/Rssss-...............#/",
+        "/*****############:####/",
+        "/*''''h''a#ZZZ#''x''''#/",
+        "/*'''ddo'f#ZZZ-'''''''w/",
+        "/*'''''''f#ZZZ#'''''''#/",
+        "/*''''''''#####'''''''w/",
+        "/*''''''''G'''G'''''''#/",
+        "/###''''''#####'''''''w/",
+        "/#S#''''''#ZZZ#'''''''#/",
+        "/#i+''''''#ZZZ-''Z##+##/",
+        "/#T#''''''#ZZZ#'',,Z''#/",
+        "4#################Z#Z##/",
+        "////////////////////////"
+      ],
+      "palettes": [ "deserter_city" ],
+      "terrain": {
+        "'": "t_floor",
+        "h": "t_floor",
+        "d": "t_floor",
+        "f": "t_floor",
+        "a": "t_floor",
+        "o": "t_floor",
+        "x": "t_floor",
+        ":": "t_door_metal_elocked",
+        "*": "t_ballistic_glass",
+        "%": [ "t_door_metal_o", [ "t_door_metal_interior_locked", 2 ] ]
+      },
+      "furniture": { "o": "f_desk" },
+      "items": {
+        "c": { "item": "science", "chance": 20, "repeat": [ 1, 2 ] },
+        "Q": { "item": "guns_energy", "chance": 100 },
+        "o": { "item": "guns_pistol_milspec", "chance": 80 }
+      },
+      "place_loot": [ { "item": "UPS_off", "x": 14, "y": 6, "chance": 85 } ],
+      "place_monster": [
+        { "group": "GROUP_MUTANT_EVOLVED", "x": 21, "y": 4 },
```

Comment:
Absolutely no mutants in cages in the middle of a city centre. Why would any group move them to a location like this, how did they get them up the stairs? How do they feed them?

Reply:
A good replacement would be some of the mutated small animals from inside the TCL, like iridescent cats.

Reply:
> A good replacement would be some of the mutated small animals from inside the TCL, like iridescent cats.

Iridescent cats have been made to no longer be mutants to fit the lore better. So nah, they wouldn't work nowadays either.

Reply:
Just regular rats then.